project-koku/koku
# Pin unleash version < 5

The unleash client is currently unable to reach the server. Pinning to an old version fixes the issue.

Changes proposed in this PR:
* Pin Unleash client < 5

Testing Instructions:
1. deploy with bonfire, and do _not_ see `Unable to get feature flag toggles, using cached provisioning`
